.github/workflows/windows.yml

```diff
@@ -276,6 +276,7 @@ jobs:
         run: |
           $ErrorActionPreference = 'Stop'
           python -m pip install -r python/requirements.txt
+          python -m pip install -r python/requirements_gui.txt
```

Comment:
In practice the difference between `open3d` and `open3d[standard]` is gui packages, so naming feels fine.
But semantically this feels a bit inconsistent: open3d + gui = open3d[standard].